components/script/script_runtime.rs

```diff
@@ -530,6 +530,11 @@ unsafe fn new_rt_and_cx_with_parent(
 
     // Enable or disable the JITs.
     let cx_opts = &mut *ContextOptionsRef(cx);
+    JS_SetGlobalJitCompilerOption(
+        cx,
+        JSJitCompilerOption::JSJITCOMPILER_BASELINE_INTERPRETER_ENABLE,
+        pref!(js.baseline.enabled) as u32,
```

Comment:
I've made  `js.baseline.enabled` control both the baseline JIT and baseline interpreters. Let me know if we should have separate prefs for both for e.g `js.baseline.jit.enabled` and `js.baseline.interpreter.enabled`

Reply:
I think it might make sense to have a separate pref for this, as it seems the baseline JIT and baseline interpreter are two different things: https://hacks.mozilla.org/2019/08/the-baseline-interpreter-a-faster-js-interpreter-in-firefox-70/

Reply:
Sure, do we want to keep the current `js.baseline.enabled` or change it to `js.baseline.jit.enabled`?

Reply:
Oh, let's change it. It was probably created before the baseline interpreter existed. How about this:

- `js.baseline-jit.enabled`
- js.baseline-interpreter.enabled`

Reply:
I've used underscores instead of dashes since that is consistent with the existing nested option `eager_unsafe_compilation` under `js.baseline` and also avoids the `serde(rename)` attributes in config/prefs.js since we'll anyway need underscores in rust syntax.